help/settings/base.py

```
@@ -207,6 +208,7 @@
 GOOGLE_TAG_MANAGER_ID = env.str('GOOGLE_TAG_MANAGER_ID')
 GOOGLE_TAG_MANAGER_ENV = env.str('GOOGLE_TAG_MANAGER_ENV', '')
 UTM_COOKIE_DOMAIN = env.str('UTM_COOKIE_DOMAIN', '')
+PRIVACY_COOKIE_DOMAIN = env.str('PRIVACY_COOKIE_DOMAIN', '')
```

I think this should be a required value. it it's not set the js cookie creation will be funky: it will try to create a cookie for a domain called `''` and will be all "que!?" but fail silently because javascript.